package.json

```diff
@@ -1,30 +1,59 @@
 {
   "name": "cookie-policy",
   "version": "1.0.5",
-  "description": "A script and stylesheet that displays the Canonical cookie policy",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "description": "A script and style sheet that displays a cookie policy notification",
+  "main": "index.html",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/canonical-webteam/cookie-policy.git"
   },
-  "author": "Anthony Dillon",
+  "author": "Anthony Dillon <anthony.dillon@canonical.com>",
+  "contributors": [
+    "Barry McGee <barry.mcgee@canonical.com>"
+  ],
   "license": "ISC",
   "bugs": {
     "url": "https://github.com/canonical-webteam/cookie-policy/issues"
   },
   "homepage": "https://github.com/canonical-webteam/cookie-policy#readme",
   "devDependencies": {
-    "babel-preset-es2015": "^6.13.2",
-    "gulp": "^3.9.1",
-    "gulp-babel": "^6.1.2",
-    "gulp-jshint": "^2.0.1",
-    "gulp-sass": "^2.3.2",
-    "gulp-sass-lint": "^1.2.0",
+    "del": "^3.0.0",
+    "gulp-eslint": "^5.0.0",
+    "gulp-sass-lint": "^1.4.0",
```

Comment:
Do we use these gulp plugins?

Reply:
Yes, they were being used as they still provided eslint and sass-lint but I've now swapped them out with their directs deps instead